pep-0690.rst

```diff
@@ -461,12 +769,13 @@ Deep eager-imports override
 ---------------------------
 
 The proposed ``importlib.eager_imports()`` context manager and
-``importlib.set_eager_imports()`` override both have shallow effects: they only
-force eagerness for the location where they are applied, not transitively. It
-would be possible (although not simple) to provide a deep/transitive version of
-one or both. That idea is rejected in this PEP because experience with the
-reference implementation has not shown it to be necessary, and because it
-prevents local reasoning about laziness of imports.
+``importlib.enable_lazy_imports(eager=...)`` override both have shallow effects:
```

Comment:
That's an interesting point that maybe should be mentioned explicitly above: `importlib.set_eager_imports()` affects _global_ interpreter state and so should be called before any threads are started.

Reply:
It's not necessarily a problem to call it after a thread has been started, but it will affect all threads running in that interpreter. I did add a clarification early in Specification that it is per-interpreter.